Library/Formula/ditaa.rb

```diff
@@ -1,13 +1,50 @@
 class Ditaa < Formula
   desc "Convert ASCII diagrams into proper bitmap graphics"
   homepage "http://ditaa.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/ditaa/ditaa/0.9/ditaa0_9.zip"
-  sha256 "d689e933b80b065cd7c349e489cfb8feea69dd3e91ca78931edc6fa6e098e689"
+  url "https://github.com/stathissideris/ditaa/archive/v0.10.tar.gz"
+  sha256 "82e49065d408cba8b323eea0b7f49899578336d566096c6eb6e2d0a28745d63b"
 
-  bottle :unneeded
+  depends_on :ant => :build
+  depends_on :java
+
+  # 0.10 Release didn't still calls itself 0.9
+  # Reported upstream at https://github.com/stathissideris/ditaa/issues/14
+  patch :DATA
```

Comment:
I'd like to wait and see upstream acknowledges this before we ship it. It may also be the reason there's no `.jar` yet.